controllers/api/webhook.go

```diff
@@ -0,0 +1,117 @@
+package api
+
+//TODO
+
+import (
+  "net/http"
+  "strconv"
+  "encoding/json"
+
+  "github.com/gophish/gophish/models"
+  "github.com/gorilla/mux"
+  log "github.com/gophish/gophish/logger"
+  "github.com/gophish/gophish/webhook"
+)
+
+func (as *Server) Webhooks(w http.ResponseWriter, r *http.Request) {
+  switch {
+  case r.Method == "GET":
+    whs, err := models.GetWebhooks()
+    if err != nil {
+      log.Error(err)
+      JSONResponse(w, models.Response{Success: false, Message: err.Error()}, http.StatusInternalServerError)
+      return
+    }
+    JSONResponse(w, whs, http.StatusOK)
+
+  case r.Method == "POST":
+    wh := models.Webhook{}
+    err := json.NewDecoder(r.Body).Decode(&wh)
+    if err != nil {
+      JSONResponse(w, models.Response{Success: false, Message: "Invalid JSON structure"}, http.StatusBadRequest)
+      return
+    }
+    err = models.PostWebhook(&wh)
+    if err != nil {
+      JSONResponse(w, models.Response{Success: false, Message: err.Error()}, http.StatusBadRequest)
+      return
+    }
+    JSONResponse(w, wh, http.StatusCreated)
+  }
+}
+
```

Comment:
Please add a docstring here in the format `Webhook ...`, such as `Webhook handles the CRUD operations for managing existing individual Gophish webhooks` or similar.